components/heatmap/HeatMap.vue

```diff
@@ -221,7 +221,14 @@ export default {
     }),
     ...mapState("playback", ["playback_state"]),
     metric_names: function () {
-      return Object.keys(this.well_values);
+      // Without this sorting the values will be unpredictable
```

Comment:
Do we really need to do this? Can we just set the default for `metric_selection_idx` to whichever idx freq is at?